src/go/protos/protos.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protos
+
+//go:generate go install google.golang.org/protobuf/cmd/protoc-gen-go
+//go:generate go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+//go:generate protoc --go_out=. --go_opt=Msctpd.proto=magma/sctpd;sctpd -I ../../../lte/protos sctpd.proto
```

Comment:
Might be worth looking into using prototool to handle our proto generation.
It takes a config based approach which might make it a bit cleaner than the go:generate commands
Other fun part is supports linting your protos
https://github.com/uber/prototool#prototool-generate

Reply:
Ah cool. Looks like there is also https://github.com/bufbuild/buf

Also, this is a temp/stop-gap while we figure out our bazel stuff cc: @themarwhal

Reply:
with the rest of magma we generate the protos using `build.py -g` command on  `~/magma/orc8r/cloud/docker`. The process it follows is 

- It runs make `make fullgen` inside a build pod [here](http://github.com/magma/magma/blob/cdc9dd678ccb35e9dd24c8b9a4aefc2894444dba/orc8r/cloud/docker/build.py#L88-L88)
- `fullgen` runs `gen_protos` which runs `make gen_protoss` in all the Makefiles under `/cloud/go` [here](http://github.com/magma/magma/blob/cdc9dd678ccb35e9dd24c8b9a4aefc2894444dba/orc8r/cloud/Makefile#L117-L117)
-  In feg this Maklefile imports a common library [here](http://github.com/magma/magma/blob/cdc9dd678ccb35e9dd24c8b9a4aefc2894444dba/feg/cloud/go/Makefile#L2-L2)
- The `make gen` [here](http://github.com/magma/magma/blob/cdc9dd678ccb35e9dd24c8b9a4aefc2894444dba/orc8r/cloud/go/module.mk#L32-L32) just do a `go generate ./...` and drops the pb files under the proto folder 


Note protos are shared between gateways, so it would be good we had a single point to generate them as we do right now